csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -206,32 +209,108 @@ private void CanRunInferenceOnAModel(GraphOptimizationLevel graphOptimizationLev
                         validateRunResults(results);
                     }
                 }
+
+                // Run inference with pinned inputs and empty outputs
```

Comment:
"empty outputs" seems  abit misleading -> I guess you mean outputs returned (`DisposableNamedOnnxValue`) ?